### PR TITLE
Organization CLI

### PR DESCRIPTION
**Pull request by macpie**
Issue https://github.com/helium/router/issues/405

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,43 +142,34 @@ device xor
 ##### Options
 XOR will only happen if `--commit` is used, otherwise it will be a dry run.
 
-### DC Tracker `dc_tracker`
+### Oraganization `oraganization`
```
Comment (michaeldjeffrey) [inline README.md:145]:
```suggestion
### Organization `organization`
```

File: README.md [hunk 2]
```diff
@@ -142,43 +142,34 @@ device xor
...
 
 #### All Orgs
 ```
-dc_tracker info all [--less 42] [--refetch]
+oraganization info all [--less 42] [--more 42]
```
Comment (michaeldjeffrey) [inline README.md:149]:
```suggestion
organization info all [--less 42] [--more 42]
```

File: README.md [hunk 3]
```diff
@@ -142,43 +142,34 @@ device xor
...
 
-`--refetch`
-Update Router information about organizations in list.
+`--more <amount>`
+Filter to Organizations that have a balance more than `<amount>`.
 
 #### Info for 1 Org
 ```
-dc_tracker info <org_id> [--refetch]
+oraganization info <org_id>
```
Comment (michaeldjeffrey) [inline README.md:160]:
```suggestion
organization info <org_id>
```

File: README.md [hunk 4]
```diff
@@ -142,43 +142,34 @@ device xor
...
 ```
-##### Options
-`--refetch`
-Update Router information about this organization.
 
-#### Refill Org Balance
+#### Update Org Balance
 ```
-dc_tracker refill <org_id> balance=<balance> nonce=<nonce>
+oraganization update <org_id> -b <balance> [--commit]
```
Comment (michaeldjeffrey) [inline README.md:165]:
```suggestion
organization update <org_id> -b <balance> [--commit]
```

File: src/apis/router_console_api.erl [hunk 5]
```diff
@@ -157,6 +160,70 @@ get_org(OrgID) ->
...
+    Url = <<Endpoint/binary, "/api/router/organizations/manual_update_router_dc">>,
+    lager:debug("get ~p", [Url]),
+    Opts = [
+        with_body,
+        {pool, ?POOL},
+        {connect_timeout, timer:seconds(2)},
+        {recv_timeout, timer:seconds(2)}
+    ],
+    Body = #{
+        organization_id => OrgID,
```
Comment (michaeldjeffrey) [inline src/apis/router_console_api.erl:200]: Do these need to go into our cache?

File: src/cli/router_cli_organization.erl [hunk 6]
```diff
@@ -0,0 +1,293 @@
...
+                    {datatype, boolean}
+                ]}
+            ],
+            fun update_cmd/3
+        ]
+    ].
+
+update_cmd(["organization", "update", OrgStr], [], Flags) ->
+    OrgID = erlang:list_to_binary(OrgStr),
+    ok = router_console_api:evict_org_from_cache(OrgID),
```
Comment (michaeldjeffrey) [inline src/cli/router_cli_organization.erl:210]: Do we want to evict from cache on dry runs?

File: src/apis/router_console_api.erl (hunk 5 above)
Comment (macpie) [reply src/apis/router_console_api.erl:200]: They get picked up on the other side by the websocket. Once the API made the update it sends us a message telling us about the update there

**Review by michaeldjeffrey — APPROVED** [review]

**Pull request merged**